content/en/docs/guides/_index.md

```diff
@@ -3,6 +3,11 @@ title: Guides
 description: Task-oriented walkthroughs of common use cases
 weight: 4
 nav_children: pages
+# menu: {main: {weight: 20, parent: Docs}}
+# menus: main
+menu:
+  main: {weight: 2}
 ---
+<iframe width="560" height="315" src="https://www.youtube.com/embed/cSGBbwvW1y4?si=bLvKqHMNG71yjXAD" title="YouTube video player" enablejsapi="1" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
Hugo provides support for embedding YouTube videos using short-codes. Use that unless there's some feature missing in it.

See https://gohugo.io/content-management/shortcodes/#youtube